### PR TITLE
Update Migros-Bank.xml

### PR DESCRIPTION
**Pull request by boleros**
Some cert issues were fixed, some hosts are gone.

### CODE REVIEW

**Comment by koops76** [comment]
@boleros See https://github.com/EFForg/https-everywhere/blob/master/ruleset-testing.md.

File: src/chrome/content/rules/Migros-Bank.xml [hunk 1]
```diff
@@ -1,42 +1,35 @@
 <!--
 	For other Migros coverage, see Migros.xml.
-
-	Mismatch:
-		- blog.bancamigros.ch
-		- p2p.bancamigros.ch
-		- blog.banquemigros.ch
-		- p2p.banquemigros.ch
```
Comment (J0WI) [inline src/chrome/content/rules/Migros-Bank.xml]: Please keep those comments, they are still up to date.

Comment (J0WI) [inline src/chrome/content/rules/Migros-Bank.xml]: The p2p domain is still mismatching too.

Comment (boleros) [inline src/chrome/content/rules/Migros-Bank.xml]: OK, I will let get back to you in a few days, when the hosts are actually gone.

Comment (J0WI) [inline src/chrome/content/rules/Migros-Bank.xml]: Are you involved? Then you should have a look at preloaded HSTS:
https://github.com/EFForg/https-everywhere/blob/master/CONTRIBUTING.md#hsts-preloaded-rules
https://hstspreload.org/

Comment (J0WI) [inline src/chrome/content/rules/Migros-Bank.xml]: Seems like they are gone now.

**Pull request merged**